### ISSUE TITLE
BC Fossil Sample - mandatory attributes to mark

### ISSUE THREAD

**Issue by LeannePyle** [bcgov/BCHeritage#437]
--need to discuss how much to make mandatory, especially if age and stratigraphy are initially unknown (what if they check uncertain?)

Fossil Common Name
Fossil Size Category
Fossil Abundance
Repository or Institution Number
Repository Name

**Comment by bferguso** [comment]
@LeannePyle - Many of these are blocked by current framework limitations. Please create a separate issue that can be addressed later.
